packages/app/src/components/inspector/ColorSelectRow.tsx

```diff
@@ -0,0 +1,191 @@
+import { memo, useMemo, useCallback } from 'react';
+import type Sketch from '@sketch-hq/sketch-file-format-ts';
+import type FileFormat from '@sketch-hq/sketch-file-format-ts';
+import {
+  Alpha,
+  ColorModel,
+  ColorPicker,
+  equalColorObjects,
+  hsvaToRgba,
+  Hue,
+  RgbaColor,
+  rgbaToHsva,
+  Saturation,
+} from 'noya-colorpicker';
+import {
+  InputField,
+  Label,
+  LabeledElementView,
+  Spacer,
+} from 'noya-designsystem';
+import styled from 'styled-components';
+
+const Row = styled.div(({ theme }) => ({
+  flex: '1',
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+}));
+
+const Column = styled.div(({ theme }) => ({
+  flex: '1',
+  display: 'flex',
+  flexDirection: 'column',
+}));
+
+const FullContent = styled.div(({ theme }) => ({
+  width: '240px',
```

Comment:
Let's remove the width here, since it was only needed for the popup color picker. This one can fill its parent. Also, padding in this case I think should be `10px 0`, since we don't need more horizontal padding.